d2lbook/translate.py

```diff
@@ -9,14 +9,29 @@
 
 def translate():
     parser = argparse.ArgumentParser(description='Translate to another language')
-    parser.add_argument('filename', nargs='+', help='the markdown files to activate')
+    # Example usage: d2lbook translate --commit 35a64ab chapter_optimization chapter_computer-vision/anchor.md
+    parser.add_argument('name', nargs='+', help='chapter dirs or markdown files to activate')
     parser.add_argument('--commit', default='latest', help='the commit of the base repo')
     args = parser.parse_args(sys.argv[2:])
 
     cf = config.Config()
     trans = Translate(cf, args.commit)
-    for fn in args.filename:
-        trans.translate(fn)
+    for name in args.name:
+        # Check if name is a file or a chapter dir
+        if not name.endswith(".md"):
+            chap_name = name
```

Comment:
Thanks for being more readable. To balance between readability and verbosity, let's keep using `source` (aka `name`) here because its semantics can be told from the if statement.